src/runtime/meter-toast-api.js

```diff
@@ -43,21 +44,31 @@ export class MeterToastApi {
     /** @private @const {!../components/dialog-manager.DialogManager} */
     this.dialogManager_ = deps.dialogManager();
 
+    const iframeArgs = this.activityPorts_.addDefaultArguments(
+      feArgs({
+        isClosable: true,
+        publicationId: deps.pageConfig().getPublicationId(),
+        productId: deps.pageConfig().getProductId(),
+        hasSubscriptionCallback: deps.callbacks().hasSubscribeRequestCallback(),
+      })
+    );
     /** @private @const {!ActivityIframeView} */
     this.activityIframeView_ = new ActivityIframeView(
       this.win_,
       this.activityPorts_,
       feUrl('/metertoastiframe'),
-      feArgs({
-        publicationId: deps.pageConfig().getPublicationId(),
-        productId: deps.pageConfig().getProductId(),
-        hasSubscriptionCallback: deps.callbacks().hasSubscribeRequestCallback(),
-      }),
+      iframeArgs,
```

Comment:
this.activityPorts_.addDefaultArguments({
  isClosable: true,
  hasSubscriptionCallback: deps.callbacks().hasSubscribeRequestCallback(),
})

Reply:
Added this with feArgs in between.

Reply:
drop feArgs.  Default args takes care of it

Reply:
Done!